tests/10apidoc/30room-create.pl

```diff
@@ -461,6 +461,6 @@ sub matrix_create_room_synced
                $_[0]->{event_id} eq $event_id
             });
          },
-      )->then_done( $room_id );
+      )->then_done( $room_id, $room_alias );
```

Comment:
please keep the documentation on the function definition in sync with this change.